src/gui/editors/PianoRoll.cpp

```diff
@@ -4590,6 +4590,43 @@ int PianoRoll::quantization() const
 	return DefaultTicksPerBar / Quantizations[m_quantizeModel.value() - 1];
 }
 
+void PianoRoll::strumNotesDn() //down-top
+//Method for chord-strumming.
+//The strum-distance is the chosen Q-value
+{
+  if( ! hasValidPattern() )
+  {
+     return;
+  }
+  int notesInChord = 0; //initializations
+  int index = 0;
+  m_pattern->addJournalCheckPoint();
```

Comment:
`m_pattern` has been renamed to `m_midiClip`

Reply:
Ok i will fix that too

Reply:
Fixed